torchvision/models/regnet.py

```diff
@@ -625,6 +640,21 @@ class RegNet_Y_32GF_Weights(WeightsEnum):
             "acc@5": 98.362,
         },
     )
+    IMAGENET1K_SWAG_LINEAR_V1 = Weights(
+        # This model has trunk weight from weakly supervised learning described in https://arxiv.org/pdf/2201.08371.pdf
+        # with only linear head finetuned to IMAGENET1K dataset
+        # This model is suitable for users that want to fine tune the pretrained trunk on other downstream datasets
```

Comment:
We copy-paste a lot across files. For the time being, until the recipe doc is completed, we could add this info once on this PR description as stated above.